### PR TITLE
Release 1.2.2

### PR DESCRIPTION
**Pull request by immel-f**
Changelogs and version bump for a ROS / PyPI release for all the recent changes

### CODE REVIEW

**Comment by immel-f** [comment]
@poggenhans I once again require your approval :sweat_smile:

File: lanelet2_python/CHANGELOG.rst
Comment (poggenhans) [inline lanelet2_python/CHANGELOG.rst:9]: Can we please remove personal emails from the changelog? They are contained in the commits anyways, but the changelog makes it a bit too simple to find them.

**Review by poggenhans — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
